docs/tutorials/github-keys.md

```diff
@@ -0,0 +1,68 @@
+---
+id: github-keys
+title: GitHub Keys
+sidebar_label: GitHub Keys 
+---
+
+## Why we need GitHub Keys?
+
+Let's first defined GitHub as a browser-based online control system of distributed version for those software developers using the Git revision control system. Git uses SSH public key authentication and not a password authetication.  Each user of the system has to generate a key. 
+
+The `SSH` is also known as Secure Shell or Secure Socket Shell [More About SSH in GitHub](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/about-ssh). SSH keys are used to authenticate secure connections they are generate through a public key cryptographic algorithm, the most common being RSA or DSA. This is a one-way formula that ensures the public key can be derived from the private key but the private key cannot be derived from the public key.
+
+
+## How to generate a new SSH key?
+
+This guide is for Windows users , more information for Linux and Mac [on this site](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
```

Comment:
In this phrase, "This guide is for Windows users ," there is an extra empty space, before the comma mark.